packages/core/src/__tests__/__snapshots__/caip10-link.test.ts.snap

```diff
@@ -274,6 +274,7 @@ Object {
       "0x0544dCF4Fce959C6C4F3b7530190cb5E1BD67cB1@eip155:1",
     ],
     "family": "caip10-eip155:1",
+    "forbidControllerChange": true,
```

Comment:
This should not be here. It will break all existing caip10-links. This doctype doesn't allow controller change anyway,

Reply:
Good call, fixed